frontend/amundsen_application/static/.betterer.results

```diff
@@ -563,6 +563,10 @@ exports[`eslint`] = {
     "js/ducks/lastIndexed/sagas.ts:1498244597": [
       [7, 2, 29, "\'action\' is defined but never used.", "566797395"]
     ],
+    "js/ducks/lineage/api/v0.ts:63450243": [
+      [19, 13, 26, "Expected the Promise rejection reason to be an Error.", "875716520"],
+      [44, 13, 26, "Expected the Promise rejection reason to be an Error.", "875716520"]
+    ],
```

Comment:
These are new, any way of fixing them?

Reply:
The results file already has a few such errors ignored. `Expected the Promise rejection reason to be an Error.`, so I am assuming this is a known issue.

Reply:
I see...
We need to review this in detail, we might want to get rid of the eslint rule if we are OK with this.